tomcat-base/src/shared/main/java/eu/xenit/alfresco/tomcat/embedded/alfresco/AlfrescoTomcatCustomizer.java

```diff
@@ -0,0 +1,24 @@
+package eu.xenit.alfresco.tomcat.embedded.alfresco;
```

Comment:
I think that this class should be under the tomcat package too same for shareTomcatCustomizer